File: MyStagePass.Model/Models/Country.cs
using System.Collections.Generic;

namespace MyStagePass.Model.Models
{
    public class Country
    {
        public int CountryID { get; set; }
        public string Name { get; set; }
		public bool IsActive { get; set; } = true;
		public virtual ICollection<City> Cities { get; set; } = new List<City>();
	}
}

File: UI/mystagepass_admin/lib/models/Country/country.dart
import 'package:json_annotation/json_annotation.dart';
import '../City/city.dart';

part 'country.g.dart';

@JsonSerializable()
class Country {
  @JsonKey(name: 'countryID')
  int? countryId;
  String? name;
  bool? isActive;
  List<City>? cities;

  Country({this.countryId, this.name, this.isActive, this.cities});

  factory Country.fromJson(Map<String, dynamic> json) =>
      _$CountryFromJson(json);
  Map<String, dynamic> toJson() => _$CountryToJson(this);
}
